Sources/Paywalls/Events/Networking/PaywallHTTPRequestPath.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PaywallHTTPRequestPath.swift
+//
+//  Created by Nacho Soto on 9/5/23.
+
+import Foundation
+
+extension HTTPRequest.PaywallPath: HTTPRequestPath {
+
+    static let serverHostURL = URL(string: "https://api-paywalls.revenuecat.com")!
+
+    var authenticated: Bool {
+        switch self {
+        case .postEvents:
+            return true
+        }
```

Comment:
I'm sure this makes sense, but why do all of these do switch self case .postEvents return hardcoded value? why not just return the harcoded value?

Reply:
So that when/if we add a new path the compiler fails and forces us to implement these for the new `case`.